recipes/bigdft/run_test.sh

```diff
@@ -0,0 +1,2 @@
+source bigdftvars.sh
+python test.py
```

Comment:
Maybe a few of the many CLI binaries being shipped? Do any of them accept `--help` or `--version` and return?

Reply:
Thanks. I've added some tests for the CLI apps I know are more regularly used with a `-h` option. I might talk to the other developers about whether all of the helper binaries should be built by default in the future.